spec/system/users_can_only_be_signed_in_to_one_type_of_account_spec.rb

```diff
@@ -0,0 +1,124 @@
+require "rails_helper"
+
+RSpec.describe "Users can only be signed in to one type of account" do
+  let!(:jobseeker) { create(:jobseeker, email: "jobseeker@example.com") }
+  let!(:publisher) { create(:publisher, dsi_data: dsi_data) }
+
+  let(:school) { create(:school, urn: "110627") }
+  let(:dsi_data) do
+    { "school_urns" => [school.urn], "trust_uids" => [], "la_codes" => [] }
+  end
+
+  before do
+    allow(JobseekerAccountsFeature).to receive(:enabled?).and_return(true)
+  end
+
+  context "when a jobseeker is signed in" do
+    before do
+      login_as(jobseeker, scope: :jobseeker)
+    end
+
+    context "when email fallback is disabled" do
+      let(:dsi_email_address) { Faker::Internet.email }
+
+      before(:each) do
```

Comment:
is `before(:each) do` not the same as `before do`?

Reply:
Yes it is – I seem to have managed to copypasta my way into using both `before` and `before(:each)` here 🍝 

I have a mild preference for explicitly stating `:each` normally, but I can see we mix and match throughout TV. Do we have any preference?

Reply:
I've changed them all to `:each` 🦘